packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -0,0 +1,40 @@
+import { ApiClient } from '../.'
+import BigNumber from 'bignumber.js'
+
+/**
+ * Loan RPCs for DeFi Blockchain
+ */
+export class Loan {
+  private readonly client: ApiClient
+
+  constructor (client: ApiClient) {
+    this.client = client
+  }
+
+  /**
+   * Creates a loan scheme transaction.
+   *
+   * @param {number} minColRatio
+   * @param {BigNumber} interestRate
+   * @param {CreateLoanSchemeOptions} options
+   * @param {string} options.id
```

Comment:
Short description of params needed such as `Unique identifier of the loan scheme`